src/App.js

```diff
@@ -1,10 +1,108 @@
-import './styles.css';
+import "./styles.css";
 
 export default function App() {
   return (
-    <div className='App'>
+    <div className="App">
       <h1>Hello From F1 Around The World</h1>
       <h2>Start editing to see some magic happen!</h2>
     </div>
   );
 }
+
+// --------------------- Everything following is proof of concept/demo ------------
+
+// urls for the ergast API, fetching up to 1000 data points
+const races_url = "http://ergast.com/api/f1.json?limit=1000";
+const drivers_url = "http://ergast.com/api/f1/drivers.json?limit=1000";
+const constructors_url = "http://ergast.com/api/f1/constructors.json?limit=300";
+
+// downloaded files linked
+const races_file = "./data/races.json";
+const drivers_file = "./data/drivers.json";
+const constructors_file = "./data/constructors.json";
+
+// deprecated, now in hooks folder
```

Comment:
If it's deprecated and its not in main no need to add it. I would remove it or comment it out